### PR TITLE
Docs: expanding `minSpareCols` and `minSpareRows` API ref (#8537)

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR expands API ref descriptions of the following options:

- `minSpareCols`
- `minSpareRows`

[skip changelog]

### CODE REVIEW

File: dist/handsontable.full.js [hunk 1]
```diff
@@ -90184,8 +90184,14 @@ var _default = function _default() {
     maxCols: Infinity,
```
Comment (swistach) [inline dist/handsontable.full.js:90184]: I'm not sure if I mentioned that before (sorry if not🙈).
We do not push modified files in `dist/` on feature branches (you can read more about contributing in https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md).

Would you mind reverting changes in this file?

File: dist/handsontable.js [hunk 2]
```diff
@@ -35137,8 +35137,14 @@ var _default = function _default() {
     maxCols: Infinity,
```
Comment (swistach) [inline dist/handsontable.js:35137]: I have the same request as for the `dist/handsontable.full.js` file.

**Review by swistach — CHANGES_REQUESTED** [review]
New descriptions are great 👏
If you revert changes in `dist/`, I think this PR will be ready to merge.

File: dist/handsontable.full.js (hunk 1 above)
Comment (kirszenbaum) [reply dist/handsontable.full.js]: @swistach - my bad, I read the contributing guidelines and build docs but forgot ;) Reverted now.

File: dist/handsontable.js (hunk 2 above)
Comment (kirszenbaum) [reply dist/handsontable.js]: @swistach - my bad, I read the contributing guidelines and build docs but forgot ;) Reverted now.

**Review by swistach — APPROVED** [review]
Now it looks good 💪

**Review by aninde — APPROVED** [review]
👍

**Pull request merged**